components/Common/Breadcrumbs/BreadcrumbLink/index.module.css

```
@@ -1,4 +1,7 @@
 .link {
+  @apply w-full 
+  truncate;
```

```suggestion
    truncate;
```

I forgot to mention that we also add extra spaces for items from following lines (as you can see on the other lines hehe)

---

haha will commit in a minute

---

I already did it 🫡